### PR TITLE
module: Add EBS extra volume support

### PR DESCRIPTION
**Pull request by jieyu**
Allow extra EBS volumes to be created for each AWS instance.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -24,12 +24,44 @@
...
+    "/dev/xvdi",
+    "/dev/xvdj",
+    "/dev/xvdk",
+    "/dev/xvdm",
+    "/dev/xvdn",
+    "/dev/xvdo",
+    "/dev/xvdp",
+    "/dev/xvdq",
+    "/dev/xvdr",
+    "/dev/xvds",
```
Comment (fatz) [inline main.tf]: ten block devices... so either we document the limitation here or make the device_name part of extra_volumes. Where device_names should be mandatory.

File: variables.tf [hunk 2]
```diff
@@ -41,6 +41,11 @@ variable "root_volume_type" {
   default     = "gp2"
 }
 
+variable "extra_volumes" {
+  description = "Extra volumes for each instance"
```
Comment (fatz) [inline variables.tf:45]: if we specify `ebs_name_format` as variable i would suggest naming this `extra_ebs_volumes`

both new variables need to be specified in the variables sheet

**Review by fatz — CHANGES_REQUESTED** [review]
please also have a look at the CI runs. Looks like you have to apply `terraform fmt`

File: main.tf (hunk 1 above)
Comment (jieyu) [reply main.tf]: Sure. I'll let users specify it.

File: variables.tf (hunk 2 above)
Comment (jieyu) [reply variables.tf:45]: I'll use `extra_volumes` for consistency as I saw `root_volume_size` and `root_volume_type` are there. I'll rename `ebs_name_format` to `extra_volume_name_format`

**Review by fatz — APPROVED** [review]

**Comment by fatz** [comment]
👍  this is safe to merge. Checked with existing cluster

**Pull request merged**